leakcanary-android-core/src/main/res/layout/leak_canary_about_screen.xml

```diff
@@ -12,6 +12,19 @@
       android:paddingRight="16dp"
       />
 
+  <Switch
+      android:id="@+id/leak_canary_about_heap_dump_switch_button"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_above="@+id/leak_canary_about_heap_dump_text"
+      android:paddingBottom="16dp"
+      android:paddingLeft="16dp"
```

Comment:
I think adding `android:paddingRight="16dp"` to make it aligned with the `leak_canary_about_text` will make it look better.